src/indra_cogex/apps/constants.py

```diff
@@ -2,6 +2,7 @@
 from pathlib import Path
 from typing import Union
 from indra.util.statement_presentation import db_sources, reader_sources
+from pusher import pusher
```

Comment:
This dependency should be captured in the setup config, perhaps under the `web` extras.